drafts/docker-open-liberty-raspberry-pi.adoc

```diff
@@ -1,60 +1,73 @@
 author_github: https://github.com/yasmin-aumeeruddy
 blog_description: 
-seo-title: Create Docker images with Open Liberty compatible with Raspberry Pi 
-seo-description: Run Liberty Bikes on the Raspberry Pi using ARM compatible Docker images.
+seo-title: Open Liberty in Docker on Raspberry Pi
+seo-description: 
 ---
-= Create Docker images with Open Liberty compatible with Raspberry Pi 
-It is well known that Raspberry Pi's are useful pieces of technology with lots of potential. We made use of them by deploying a kubernetes cluster to run link:https://github.com/OpenLiberty/liberty-bikes[Liberty Bikes], a real-time web based multiplayer game running on OpenLiberty. Fortunately Open Liberty is small and lightweight so it was never a problem to fit it on the original Raspberry Pi back in 2012. Although, we have used Raspberry Pi 3's for this demonstration.
+= Open Liberty in Docker on Raspberry Pi
 
-The complication when moving to a Raspberry Pi is that it has an ARM CPU at its heart, meaning that machine code compiled for x86 or x86-64 CPUs (the standard CPUs you get in laptops/desktops today) won’t execute. Liberty is written in Java so it doesn’t have to worry about what the CPU architecture is: as long as there is a Java runtime it’s able to run (as the Java runtime itself interprets the Java code and executes it on the hardware). We do have official Open Liberty images on Docker Hub, but they are layered on top of the official IBM image, which is built on top of Ubuntu and doesn’t have a JVM that will run on ARM.
+Yasmin Aumeeruddy <https://github.com/yasmin-aumeeruddy>
+
+It is well known that Raspberry Pi's are useful pieces of technology with lots of potential so I set off on a mission to see how easy it is to get a Raspberry Pi-compatible Docker image of Open Liberty. Fortunately Open Liberty is small and lightweight so it was never a problem to fit it on the original Raspberry Pi back in 2012. Although,I have used Raspberry Pi 3's for this demonstration.
 
-This means you can’t take our Open Liberty Docker image from Docker Hub, which is designed for an x86 architecture, and run it on the Pi without any configuration to the Docker images. We need to edit the Dockerfiles that define how to build the x86 Liberty Docker image and modify them to build 
-an image for the ARM Raspberry Pi.
+The complication when moving to a Raspberry Pi is that it has an ARM CPU at its heart, meaning that machine code compiled for x86 or x86-64 CPUs (the standard CPUs you get in laptops/desktops today) won’t execute. Liberty is written in Java so it doesn’t have to worry about what the CPU architecture is: as long as there is a Java runtime it’s able to run (as the Java runtime itself interprets the Java code and executes it on the hardware). We do have official Open Liberty images on Docker Hub, but they are layered on top of the official IBM image, which is built on top of Ubuntu and doesn’t have a JVM that will run on ARM.
```

Comment:
"The complication when running Docker on a Raspberry Pi is that..." - you've not talked about "moving" from one place to another prior to this paragraph so it doesn't quite make sense to do it now.

Reply:
Is the fact that the IBM image is built on top of Ubuntu the reason that it doesn't have an ARM JVM? Or should it say "is built on top of Ubuntu but doesn't have a JVM..."?